projects/kit/components/progress/progress-bar/progress-bar.component.less

```diff
@@ -82,6 +83,10 @@
             });
         }
 
+        &[data-size='xxs'] {
+            height: 0.125rem;
```

Comment:
Unfortunately, this size is not compatible with our design system 😢 
I think we should not merge this PR 🤔

Reply:
@evantrimboli 
But it not big problem. You can easily set any size you wish for `ProgressBar`.

```html
<progress
    tuiProgressBar
    max="100"
    [value]="50"
    class="progress"
></progress>

```

```css
.progress {
    height: 0.125rem;
}
```

Reply:
Ok no worries. I'll close this out.